CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 * [ENHANCEMENT] Add "integration" as a label for `cortex_alertmanager_notifications_total` and `cortex_alertmanager_notifications_failed_total` metrics. #3056
 * [ENHANCEMENT] Add `cortex_ruler_config_last_reload_successful` and `cortex_ruler_config_last_reload_successful_seconds` to check status of users rule manager. #3056
 * [ENHANCEMENT] Memcached dial() calls now have an optional circuit-breaker to avoid hammering a broken cache #3051
+* [ENHANCEMENT] Add TLS support to etcd client (used by the hash ring). #3091
```

Comment:
We normally use PR numbers in the changelog, so this should be

```suggestion
* [ENHANCEMENT] Add TLS support to etcd client. #3102
```

(Also etcd client is used at different places, eg. HA Tracker, so I've removed "used by the hash ring" part).